code/06_analysis.Rmd

```diff
@@ -179,9 +220,95 @@ df %>%
                                       name == "Experiential Avoidance" ~ scale_y_continuous(name = "Avoidance", limits = c(1, 7), breaks = 1:7))) +
   theme_classic()
 
-ggsave(filename = here("figures", "Avoidance by Identity.png"),
+ggsave(filename = here("figures", "Avoidance by Identity - Raw Data.png"),
+       width = 8,
+       height = 4,
+       units = "in",
+       dpi = 1200)
+```
+
+### Model Data
+
+```{r}
+# Calculate marginal effects; rather than use conditional_effects() plot, extract
+# plot data and create the plot manually
+plot_data_ba <- conditional_effects(
+  model_ba_control,
+  effects = "anxiety_identity",
+  method = "posterior_linpred"
+) %>%
+  pluck("anxiety_identity")
+
+plot_data_ea <- conditional_effects(
+  model_ea_control,
+  effects = "anxiety_identity",
+  method = "posterior_linpred"
+) %>%
+  pluck("anxiety_identity")
```

Comment:
In each `conditional_effects()`, add arguments `condition = data.frame(symptom_severity = 0)` (to condition exactly at 0) and `robust = FALSE` (to use mean rather than median as measure of central tendency). This way, total change shown in the plot will map directly onto total change implied by our coefficients for anxiety identity centrality in Table 2.